src/settings.ts

```diff
@@ -37,6 +37,7 @@ export const SETTINGS_DEFAULT = {
 	lookForInlineEventsInNotes: true,
 	lookForTagsForTimeline: false,
 	stylizeDateInline: false,
+        showNoteHeadersInCardBody: true,
```

Comment:
it seems your linter is not setup correctly, that also means the github actions didn't pick this up :thinking:

Reply:
note to self, get a linter! I don't have one setup and that one slipped through.
Will fix.

Reply:
Ran the linter as per your below comment, didn't seem to change it, did a copy paste from the line above, hopefully that resolves it.